packages/fetchai/skills/ml_data_provider/README.md

```diff
@@ -0,0 +1,21 @@
+# ML Train Provider
+
+## Description
+
+This skill sells ML data for training.
+
+This skill is part of the Fetch.ai ML skill demo. It registers its "ML data selling service" on the soef. It can be requested (for example by an agent with the ml_train skill) to provide specific data samples, which it delivers after it receives payment. 
```

Comment:
```suggestion
This skill is part of the Fetch.ai ML skill demo. It registers its "ML data selling service" on the SOEF. It can be requested (for example by an agent with the ml_train skill) to provide specific data samples, which it delivers after it receives payment.
```